packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -11,7 +11,11 @@ const propTypes = {
    * Custom input attributes to apply to the time input.
    */
   inputAttributes: PropTypes.object,
-    /**
+  /**
+   * Name of the time input. The name should be unique and is required.
```

Comment:
Minor:

Wouldn't the prop tables list it as required? Perhaps potential duplication documentation?